Keas.Mvc/Controllers/SystemController.cs

```diff
@@ -35,8 +35,9 @@ public IActionResult Emulate()
         public async Task<IActionResult> Emulate(EmulateUserViewModel model)
         {
             //Log.Information("Emulation attempted for {username}", username);
+            var lookupVal = model.UserEmail.Trim();
 
-            var user = await _context.Users.SingleOrDefaultAsync(u => u.Email == model.UserEmail || u.Id == model.UserEmail);
+            var user = await _context.Users.SingleOrDefaultAsync(u => u.Email.Equals(lookupVal, StringComparison.OrdinalIgnoreCase) || u.Id.Equals(lookupVal, StringComparison.OrdinalIgnoreCase));
```

Comment:
How will the second part of the OR ever be true?  Why would the user email match the u.Id value?

Reply:
@srkirkland We can lookup be kerb. This is a System Admin only page as well.

Reply:
@jSylvestre if we can lookup by kerb then why is the variable called UserEmail?

Reply:
@srkirkland Probably copy and paste. But this functionality is in more than one place. I blame @jcubbage but it might have been me.

Reply:
ok fine 🚢